root.html

```diff
@@ -1,6 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <meta http-equiv="Content-Security-Policy" content="script-src 'unsafe-inline' 'self' cdn.segment.com/analytics.js/ 'nonce-{{.Nonce}}' 'unsafe-eval'">
```

Comment:
"unsafe-inline" is ignored when "nonce" is also present. But by including it, we degrade gracefully and remain fully functional for browsers that don't support "nonce".